django_pain/management/commands/import_payments.py

```diff
@@ -0,0 +1,64 @@
+"""Command for importing payments from bank."""
+import importlib
+import sys
+
+from django.core.exceptions import ValidationError
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from django_pain.models import BankAccount
+
+
+class Command(BaseCommand):
+    """Import payments from bank."""
+
+    help = 'Import payments from the bank. Bank statement should be provided on standard input.'
+
+    def add_arguments(self, parser):
+        """Command takes one argument - dotted path to parser class."""
+        parser.add_argument('parser', type=str, help='dotted path to parser class')
+
+    @staticmethod
+    def get_parser(parser_path):
+        """Get parser instance from parser path argument."""
+        module_name, class_name = parser_path.rsplit('.', 1)
+
+        parser_module = importlib.import_module(module_name)
+        parser_class = getattr(parser_module, class_name)
+        return parser_class()
+
+    def handle(self, *args, **options):
+        """Run command."""
+        parser = self.get_parser(options['parser'])
+
+        try:
+            payments = parser.parse(sys.stdin)
+        except BankAccount.DoesNotExist as e:
+            self.stderr.write(self.style.ERROR(str(e)))
+            sys.exit(1)
+
+        for payment_parts in payments:
+            try:
+                with transaction.atomic():
+                    if type(payment_parts) is tuple:
```

Comment:
This is probably not a good condition to check if the output is nested. Lists and tuples should be interchangeable in this case. The major difference is whether the output single list or a nested one.

Reply:
I can check for both tuple and list, but it would make the output format ambiguous.

Reply:
OK, I have decided to use `Sequence` from `typing`. Fixed.